### PR TITLE
docs: document convert_images

### PR DESCRIPTION
**Pull request by giuseppe**
Closes: https://github.com/containers/storage/issues/1749

### CODE REVIEW

File: docs/containers-storage.conf.5.md [hunk 1]
```diff
@@ -106,6 +106,10 @@ containers/storage supports three keys
     Tells containers/storage where an ostree repository exists that might have
     previously pulled content which can be used when attempting to avoid
     pulling content from the container registry
+  * convert_images = "false" | "true"
+    If set to true, containers/storage will convert images to the new format in order
```
Comment (vrothberg) [inline docs/containers-storage.conf.5.md:110]: Can we be explicit on what "the new format" refers to?  Just to avoid misinterpretation.

Comment (giuseppe) [reply docs/containers-storage.conf.5.md]: sure, I'll specify what that means

**Review by vrothberg — APPROVED** [review]
LGTM

**Pull request merged**